src/main/webapp/resources/js/apis/analysis/analysis.js

```diff
@@ -149,6 +149,77 @@ export async function getSistrResults(submissionId) {
   }
 }
 
+/**
+ * Get the output file info
+ * @param {number} submissionID Submission ID
+ * @return {Promise<*>} `data` contains the OK response; `error` contains error information if an error occurred.
+ */
+export async function getOutputInfo(submissionId) {
+  try {
+    const res = await axios.get(`${URL}/${submissionId}/outputs`);
+    return res.data;
+  } catch (error) {
+    return { error };
+  }
+}
+
+/**
+ * Get the data from the output file for with the supplied chunk size
+ * @param {object} contains the output file data
+ * @return {Promise<*>} `data` contains the OK response; `error` contains error information if an error occurred.
+ */
+export async function getDataViaChunks(outputFileInfoObject) {
+  try {
+    const res = await axios.get(
+      `${URL}/${outputFileInfoObject.submissionId}/outputs/${
+        outputFileInfoObject.fileId
+      }
+            ?${$.param({
```

Comment:
NOPE!

Reply:
Fixed in f326e43b0440a94abd50d69d7ac0107d1986e12b